tests/data/short_circuit/single_phase_to_ground_c_maximum/sc_output_batch.json

```diff
@@ -2,23 +2,23 @@
   {
     "node":
       [
-        {"id": 0, "energized": 1, "u_pu": [1.1000039840999545, 1.1000016506895227, 1.1000042703666313], "u": [95263.13944946544, 95262.9373701943, 95263.16424088688], "u_angle": [-1.6752923662903967e-06, -2.094395252466148, 2.0943960267868786]},
-        {"id": 1, "energized": 1, "u_pu": [3.197510482767568e-16, 1.9056219941306507, 1.9056225158197297], "u": [1.8460835379625056e-12, 11002.113712850027, 11002.116724823329], "u_angle": [-0.4002196382694752, 3.1415256616159075, 2.0944613189550862]},
-        {"id": 2, "energized": 1, "u_pu": [0.00015708279091388333, 1.9055814205757478, 1.9057370030880023], "u": [0.9069179161918824, 11001.879461321574, 11002.777717374887], "u_angle": [0.8872079931202395, 3.1414518940151694, 2.0943744277646714]},
-        {"id": 3, "energized": 1, "u_pu": [0.0001767182112092264, 1.9055763493550697, 1.9057513147761516], "u": [1.02028306812356, 11001.850182615339, 11002.860345944942], "u_angle": [0.887206319743235, 3.141442672906039, 2.094363567002621]}
+        {"id": 0, "energized": 1, "u_pu": [1.1000039840999545, 1.1000016506895227, 1.1000042703666313], "u": [95263.13944946544, 95262.9373701943, 95263.16424088688], "u_angle": [-1.6752923662861285e-06, -2.094395252466148, 2.0943960267868786]},
+        {"id": 1, "energized": 1, "u_pu": [1.6612656886172617e-16, 1.9056219941306511, 1.9056225158197297], "u": [9.591321925186651e-13, 11002.113712850029, 11002.116724823329], "u_angle": [-0.5733177432085695, 3.1415256616159075, 2.0944613189550867]},
+        {"id": 2, "energized": 1, "u_pu": [0.00015708279091403854, 1.905581420575748, 1.9057370030880025], "u": [0.9069179161927785, 11001.879461321576, 11002.77771737489], "u_angle": [0.8872079931204931, 3.14145189401517, 2.0943744277646714]},
+        {"id": 3, "energized": 1, "u_pu": [0.0001767182112095378, 1.90557634935507, 1.9057513147761516], "u": [1.020283068125358, 11001.850182615339, 11002.860345944942], "u_angle": [0.8872063197433703, 3.1414426729060394, 2.0943635670026213]}
```

Comment:
you're not calculating things differently, right? why do you get rounding errors?

Reply:
I don't this this would be an issue. Even if you upgrade your numpy library can result in a different solution. Floating pointer is never exact.

Reply:
it shouldn't be but for the sake of keeping this PR clean i would suggest to try and keep the rounding differences to a minimum

Reply:
@mgovers I would not spend too much time on rounding differences.